entry.sh

```diff
@@ -2,19 +2,18 @@
 
 set -euo pipefail
 
-kubectl delete secret image-pull-secret 2>/dev/null || true
-
 if [ -n "${DOCKER_USERNAME:-}" ] && [ -n "${DOCKER_PASSWORD:-}" ]; then
-	echo "Creating DockerHub image-pull-secret from environment variables"
-	kubectl create secret docker-registry image-pull-secret \
-		--docker-server='https://index.docker.io/v1/' \
-		--docker-username="${DOCKER_USERNAME}" \
-		--docker-password="${DOCKER_PASSWORD}"
+  echo "Creating DockerHub image-pull-secret from environment variables"
+  kubectl create secret docker-registry image-pull-secret \
+    --docker-server='https://index.docker.io/v1/' \
+    --docker-username="${DOCKER_USERNAME}" \
+    --docker-password="${DOCKER_PASSWORD}" \
+    -o yaml --dry-run=client | kubectl apply -f -
```

Comment:
We can delete this whole block now can't we? The new approach should be used going forward.

Reply:
Yes, I removed it on a new commit :ok_hand: .